src/open_inwoner/components/templatetags/table_tags.py

```diff
@@ -98,6 +100,23 @@ def add_row_if_not_empty(key, label):
 
         rows.append(row)
 
+    if zaak:
+        rows.append(
+            {
+                "headers": [{"text": _("Zaakidentificatie")}],
+                "cells": [
+                    {
+                        "text": zaak.identificatie,
+                        "url": reverse(
+                            "cases:case_detail", kwargs={"object_id": str(zaak.uuid)}
+                        ),
+                        "hyperlink_text": _("Ga naar zaak"),
```

Comment:
Oops, I missed this one - we use 'link' in the entire project for hyperlinks, and just the one underscore would not be BEM-like, so at least use two underscores with the blockname first - probably something like `.case-detail__link`